server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/mailbox/CassandraSessionModule.java

```diff
@@ -273,7 +274,27 @@ public CassandraSchemaChecker(CassandraSchemaVersionManager versionManager) {
 
         @Override
         public void initModule() {
-            versionManager.ensureSchemaIsSupported();
+            SchemaState schemaState = versionManager.computeSchemaState();
```

Comment:
IMO logging here is something that is important it should be unit test maybe by extracting this inner class.